CouplingDataUser.H

```diff
@@ -60,6 +72,13 @@ public:
     //- Read the coupling data from the buffer
     virtual void read(double* dataBuffer, const unsigned int dim) = 0;
 
+    //- Given the meshConnectivity, return if the underlying loactionType of the
+    //- interface nodes is supported by the data set
+    virtual bool isLocationTypeSupported(const bool meshConnectivity) const = 0;
+
+    //- Get the name of the current data field
+    virtual std::string getDataName() const = 0;
```

Comment:
Is this related to this PR, or just a nice side addition?

Reply:
No it's related. Otherwise the user won't know the origin of the issue.

Reply:
Right, I did not see it in the error messages I looked at that moment, but it is in the error message of the `CouplingDataUser`.